src/main/java/gui/MonitorScene.java

```diff
@@ -140,52 +145,202 @@ private Pane createLoggerPane() {
         Image black = new Image(streamEnd.toURI().toString());
-        imageView.setImage(black);
 
-        // When menu options are clicked
-        openVideo(openVideo, primaryStage);
-        connectStream(connectStream, primaryStage);
-        theStream(theStream);
+        final int height = 300;
+        for (int i = 0; i < imageViews.size(); i++) {
+            imageViews.get(i).setImage(black);
+            imageViews.get(i).setFitHeight(height);
+            imageViews.get(i).setPreserveRatio(true);
+            imageViews.get(i).setSmooth(true);
+            imageViews.get(i).setCache(false);
+        }
+        mediaPlayerPane.getChildren().addAll(imageViews);
+    }
 
-        // Add menuPane and mediaPlayerPane to list of panes
-        ArrayList<Pane> menuMediaList = new ArrayList<>();
-        menuMediaList.add(menuPane);
-        menuMediaList.add(mediaPlayerPane);
+    /**
+     * Remove all current cameras.
+     * @param clearImageViewers the current imageViewers
+     */
+    private void resetCameras(final MenuItem clearImageViewers) {
+        clearImageViewers.setOnAction(event -> {
+            getController().closeStream();
+            mediaPlayerPane.getChildren().clear();
+        });
+    }
 
-        return menuMediaList;
+    /**
+     * Shut down application.
+     * @param closeApp menu item
+     */
+    private void closeApp(final MenuItem closeApp) {
+        closeApp.setOnAction(event -> System.exit(0));
+    }
+
+    /**
+     * Let user select a configuration file.
+     * @param configFile the file
+     * @param primaryStage the starting stage
+     */
+    private void openConfig(final MenuItem configFile, final Stage primaryStage) {
+        configFile.setOnAction(event -> {
+            FileChooser chooser = new FileChooser();
+            chooser.setTitle("Select Configuration File (JSon format)");
+            File file = chooser.showOpenDialog(primaryStage);
+            JsonHandler jsonHandler = new JsonHandler(file.toString());
+            configure(jsonHandler);
+        });
+    }
+
+    /**
+     * Configure the standard file.
+     * @param standardFile a provided standard file
+     */
+    private void standardConfig(final MenuItem standardFile) {
+        standardFile.setOnAction(event -> {
+            JsonHandler jsonHandler = new JsonHandler("files/standard.json");
+            configure(jsonHandler);
+        });
+    }
+
+    /**
+     * Load the configuration file.
+     * @param jsonHandler the current jsonHandler
+     */
+    private void configure(final JsonHandler jsonHandler) {
+        int chests = jsonHandler.getAmountChests(0);
+        int people = jsonHandler.getAmountPeople(0);
+        int cameras = jsonHandler.getCameraLinks(0).size();
+        getController().setParameters(people, chests);
+
+        for (int k = 0; k < cameras; k++) {
+            getController().createCamera(jsonHandler.getCameraLinks(0).get(k));
```

Comment:
In camera we can add how many chests there are to look for. Right now we don't have that built in in the camera handler class, but that needs to be built in so that we can add cameras with the amount of chests.

Reply:
fixed it myself.